lib/IDE/CodeCompletionResultBuilder.cpp

```diff
@@ -234,60 +234,58 @@ void CodeCompletionResultBuilder::setAssociatedDecl(const Decl *D) {
 void CodeCompletionResultBuilder::addCallArgument(
     Identifier Name, Identifier LocalName, Type Ty, Type ContextTy,
     bool IsVarArg, bool IsInOut, bool IsIUO, bool IsAutoClosure,
-    bool UseUnderscoreLabel, bool IsLabeledTrailingClosure, bool HasDefault) {
+    bool IsLabeledTrailingClosure, bool IsForOperator, bool HasDefault) {
```

Comment:
`UseUnderscoreLabel` was meant to indicate "is labeled trailing closure" when it's introduced 😅